### ISSUE TITLE
Have a pathway for submitting issues with the new Laravel website

### ISSUE THREAD

**Issue by darin-sai** [laravel/ideas#1801]
I've noticed some issues with the new Laravel docs layout, and was looking to check to see if anyone had reported it (not all items visible in the navigation sidebar). I don't see what the official place to report such issues would be.

**Comment by brunogaspar** [comment]
Have you checked https://github.com/laravel/laravel.com-next ?

I know the issues are closed, but maybe you can try and fix the issue by submitting a pull request.

Otherwise, i'm not aware of any dedicated place for website issues.

**Comment by darin-sai** [comment]
Yeah, looks like that's where I could try to identify what exactly is going on and take a stab at a fix. Don't really have a ton of time for that, so I was hoping to log it somewhere. Seemed like a big enough issue to warrant reporting.

**Comment by brunogaspar** [comment]
@driesvints Any idea if there's any place for that or if here is fine?

**Comment by jimgwhit** [comment]
On https://laravel.com/ Right side bottom there's a Github link, it's an image, not words:
That takes you to:
```ruby
https://github.com/laravel
```
You can navigate from there.

**Comment by driesvints** [comment]
Really, if it's a big issue feel free to ping anyone of the core team but other than that just send in a PR when you have time.
